examples/apiGateway.ts

```diff
@@ -0,0 +1,13 @@
+import { ApiGatewayAuthorizer, FunctionHandler } from "../src";
+
+export const handler: FunctionHandler<
+    ApiGatewayAuthorizer.Event,
+    ApiGatewayAuthorizer.Result
+> = (event) => {
+    return {
+        statusCode: 200,
```

Comment:
I believe this one will not be compiled, since `ApiGatewayAuthorizer.Result` has only `isAuthorized` and `context` properties.

Reply:
TODO for me: add PR check for building all sources and examples.

Reply:
I've added automatic check which compiles all ts. Please rebase on master.

Reply:
> I believe this one will not be compiled, since ApiGatewayAuthorizer.Result has only isAuthorized and context properties.

🤔 can't understand why this works.

Reply:
https://lemoine-benoit.medium.com/why-does-typescript-sometimes-fails-to-type-check-extra-properties-fd230ebbc295
If you want the compiler to preform excess check you have to add extra typing for the return type
```typescript
export const handler: Handler.ApiGatewayAuthorizer = (
    event
): ApiGatewayAuthorizer.Result => {
  // error
}
```